test/performance/benchmarks/reconciliation-delay/main.go

```diff
@@ -19,218 +19,150 @@ package main
 import (
 	"context"
 	"flag"
+	"fmt"
 	"log"
+	"math"
 	"os"
 	"path/filepath"
 	"time"
 
-	"github.com/google/mako/go/quickstore"
+	vegeta "github.com/tsenart/vegeta/v12/lib"
+	"k8s.io/apimachinery/pkg/types"
+	"knative.dev/pkg/injection"
+	"knative.dev/pkg/signals"
 	"sigs.k8s.io/yaml"
 
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/watch"
 	"knative.dev/pkg/apis"
 
+	netv1alpha1 "knative.dev/networking/pkg/apis/networking/v1alpha1"
+	networkingclient "knative.dev/networking/pkg/client/injection/client"
 	duckv1 "knative.dev/pkg/apis/duck/v1"
 	"knative.dev/pkg/kmeta"
 	"knative.dev/pkg/ptr"
-	"knative.dev/pkg/signals"
-
-	netv1alpha1 "knative.dev/networking/pkg/apis/networking/v1alpha1"
-	networkingclient "knative.dev/networking/pkg/client/injection/client"
-	"knative.dev/pkg/test/mako"
 	autoscalingv1alpha1 "knative.dev/serving/pkg/apis/autoscaling/v1alpha1"
 	v1 "knative.dev/serving/pkg/apis/serving/v1"
 	servingclient "knative.dev/serving/pkg/client/injection/client"
-	"knative.dev/serving/test/performance"
+	"knative.dev/serving/test/performance/performance"
 )
 
 const (
-	benchmarkName = "Development - Serving deployment probe"
+	benchmarkName = "Knative Serving reconciliation delay"
+	template      = "knative-service-template.yaml"
```

Comment:
Should we get read of this template concept and just create the ksvc in code?
I suspect values like `ghcr.io/knative/autoscale-go:latest` are not going to change any time soon.
Btw it is kind of confusing (unless you dive into the code) that some tests have a *-setup.yaml to create the ksvcs while others do it via main in the job pod.

Reply:
Good point, I just went with what was there, but yes, we should change that to do in code as well.

Reply:
Done.